packages/dd-trace/src/appsec/iast/security-controls/parser.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+const log = require('../../../log')
+const { getMarkFromVulnerabilityType, CUSTOM_SECURE_MARK } = require('../taint-tracking/secure-marks')
+
+const SECURITY_CONTROL_DELIMITER = ';'
+const SECURITY_CONTROL_FIELD_DELIMITER = ':'
+const SECURITY_CONTROL_ELEMENT_DELIMITER = ','
+
+const INPUT_VALIDATOR_TYPE = 'INPUT_VALIDATOR'
+const SANITIZER_TYPE = 'SANITIZER'
+
+const validTypes = [INPUT_VALIDATOR_TYPE, SANITIZER_TYPE]
+
+function parse (securityControlsConfiguration) {
+  const controls = new Map()
+
+  securityControlsConfiguration?.replace(/\s/g, '')
```

Comment:
too simple regex I think, what if a directory in the list has an space?

Reply:
good point. replacing it